### ISSUE TITLE
ContextualHelp with scrollbar is not keyboard accessibile

### ISSUE THREAD

**Issue by carolframpton** [adobe/react-spectrum#5303]
### Provide a general summary of the issue here

I have a ContextHelp which can be long and at times displays a vertical scrollbar.
It is not possible to get to the content at the bottom using just a keyboard.

### 🤔 Expected Behavior?

Should be able to scroll the help content with the keyboard

### 😯 Current Behavior

Can not scroll the help content with the keyboard.

### 🖥️ Steps to Reproduce

Reduce the height of your browser window.
Run https://reactspectrum.blob.core.windows.net/reactspectrum/169b0467b0c1a2638bee97d25cd432c45f27cfc8/storybook/index.html?path=/story/contextualhelp--default&providerSwitcher-express=false
and keyboard to the button. Press the button.

![image](https://github.com/adobe/react-spectrum/assets/71789937/a77e67cb-d082-45f8-98f9-0f11ab96d5e8)

### Version

3.31.0

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

### What operating system are you using?

macOs

### 🧢 Your Company/Team

Adobe/Admin Console

**Comment by LFDanLu** [comment]
Reproduced locally, problem is that we don't have a focusable element with the dialog's scrollable region. Ideally the scrollable region itself would be focusable but this is currently using Dialog which has the tabIndex applied to the div wrapping the scrollable region...

**Comment by majornista** [comment]
Chromium will be releasing Keyboard focusable scroll containers, similar to the way Firefox makes scroll containers receive focus very soon: https://chromestatus.com/feature/5231964663578624.
